migrator/migrations/4_add_grant_user_to_users.up.sql

```diff
@@ -0,0 +1 @@
+GRANT SELECT, INSERT, UPDATE  ON users TO gateway;
```

Comment:
It also should be DELETE action granted to the user.

Reply:
4_add_grant_user_to_users.down.sql